packages/flutter/test/material/elevated_button_test.dart

```diff
@@ -2178,6 +2178,98 @@ void main() {
 
     focusNode.dispose();
   });
+
+  testWidgets('Default iconAlignment', (WidgetTester tester) async {
+    Widget buildWidget({ required TextDirection textDirection }) {
+      return MaterialApp(
+        home: Directionality(
+          textDirection: textDirection,
+          child: Center(
+            child: ElevatedButton.icon(
+              onPressed: () {},
+              icon: const Icon(Icons.add),
+              label: const Text('button'),
+            ),
+          ),
+        ),
+      );
+    }
+
+    // Test default iconAlignment when textDirection is ltr.
+    await tester.pumpWidget(buildWidget(textDirection: TextDirection.ltr));
+
+    final Offset buttonTopLeft = tester.getTopLeft(find.byType(Material).last);
+    final Offset iconTopLeft = tester.getTopLeft(find.byIcon(Icons.add));
+
+    // The icon is aligned to the left of the button.
+    expect(buttonTopLeft.dx, iconTopLeft.dx - 16.0); // 16.0 - padding between icon and button edge.
+
+    // Test default iconAlignment when textDirection is rtl.
+    await tester.pumpWidget(buildWidget(textDirection: TextDirection.rtl));
+
+    final Offset buttonTopRight = tester.getTopRight(find.byType(Material).last);
+    final Offset iconTopRight = tester.getTopRight(find.byIcon(Icons.add));
+
+    // The icon is aligned to the right of the button.
+    expect(buttonTopRight.dx, iconTopRight.dx + 16.0); // 16.0 - padding between icon and button edge.
+  });
+
+  testWidgets('iconAlignment can be customized', (WidgetTester tester) async {
+    Widget buildWidget({
+      required TextDirection textDirection,
+      required IconAlignment iconAlignment,
+    }) {
+      return MaterialApp(
+        home: Directionality(
+          textDirection: textDirection,
+          child: Center(
+            child: ElevatedButton.icon(
+              onPressed: () {},
+              icon: const Icon(Icons.add),
+              label: const Text('button'),
+              iconAlignment: iconAlignment,
+            ),
+          ),
+        ),
+      );
+    }
+
+    // When the layout is ltr and iconAlignment is start, the icon is aligned to the left of the button.
+    await tester.pumpWidget(buildWidget(textDirection: TextDirection.ltr, iconAlignment: IconAlignment.start));
```

Comment:
```suggestion
    await tester.pumpWidget(buildWidget(
      textDirection: TextDirection.ltr,
      iconAlignment: IconAlignment.start,
    ));
```

This exceed the 80 characters line limit. Add line breaks in the parameter to readability and 80 charactesr limit.

Reply:
Similar fix for the tests in this class, as well as in other classes for this PR.

Reply:
Note also: there's no hard 80 character line limit however in cases like this the code is easier to read if you apply the conventional Flutter formatting rules.

Reply:
Done!